### PR TITLE
Update common-attributes.adoc to resolve formatting issues

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* No JIRA ticket created as I am trying to fix something quickly

    There is a problem in the attributes with spaces and mis-formatting causing problems, for example in the IBM attributes:

image

As you can see from the attached, the [preview](https://84869--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-ibm-cloud) for this PR looks like it resolves the issue:
    
image

See [PR for 4.15](https://github.com/openshift/openshift-docs/pull/84873) also. Thanks    
    

### Version 

* OCP 4.16

Issue:

Link to docs preview:

### QE review:
- [No QE approval as no content changed ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by anarnold97** [comment]
Please also see

* [4.15](https://github.com/openshift/openshift-docs/pull/84873)
* [4.14](https://github.com/openshift/openshift-docs/pull/84874)

File: _attributes/common-attributes.adoc [hunk 1]
```diff
@@ -237,18 +237,13 @@ endif::[]
 :coo-first: Cluster Observability Operator (COO)
 :coo-full: Cluster Observability Operator
 :coo-short: COO
-//ODF
+// ODF
```
Comment (dfitzmau) [inline _attributes/common-attributes.adoc:240]: Thanks for raising this PR @anarnold97 . I see that some other commented sections need a space. Would you be OK to correct all such instances?

Screenshot from 2024-11-13 14-22-06

Comment (dfitzmau) [reply _attributes/common-attributes.adoc:240]: Probably best to fix them all so no unwarranted error exists when the AsciiDoc interpreter scans the file.

Comment (anarnold97) [reply _attributes/common-attributes.adoc:240]: @dfitzmau - Not causing problems... it is the white spaces but fixed anyhoo

**Pull request merged**